Vagrantfile

```diff
@@ -49,6 +49,12 @@ Vagrant.configure("2") do |config|
     systemctl enable --now docker
     systemctl restart docker
     usermod -aG docker vagrant
+
+    # Firewall setup
+    ufw default allow outgoing
+    ufw default deny incoming
+    ufw allow in on eth0 to any port 22 proto tcp
+    echo "y" | ufw enable
```

Comment:
Consider setting ufw logging level to med or at minimum adding a line in the vagrantfile as a visual reminder to set specific level, as needed.

Reply:
Good thought.  After reviewing the levels it feels medium is most appropriate.  Pushed an update to add `ufw logging medium`.